README.md

```diff
@@ -62,29 +62,186 @@ latexmk
 - 在vscode上编译 (LaTex Workshop)：对于使用vscode写tex的用户，添加如下配置，可在vscode上使用XeLaTeX：
 
 ```
-"latex-workshop.latex.tools": [
-    {
-        "name": "xelatex",
-        "command": "xelatex",
-        "args": [
-            "-synctex=1",
-            "-interaction=nonstopmode",
-            "-file-line-error",
-            "%DOC%"
-        ]
-    }
-],
-"latex-workshop.latex.recipes": [
-    {
-        "name": "xelatex -> bibtex -> xelatex*2",
-        "tools": [
-            "xelatex",
-            "bibtex",
-            "xelatex",
-            "xelatex"
-        ]
-    }
-]
+// ======================== LaTeX 设置 BEGIN  ========================
+	// bibtex 格式
+	"latex-workshop.bibtex-format.tab": "tab",
+
+	// 自动编译，全部关闭，当且仅当你认为有需要的时候才会去做编译
+	"latex-workshop.latex.autoBuild.run": "never",
+	"latex-workshop.latex.autoBuild.cleanAndRetry.enabled": false,
+
+	// 设置 latex-workshop 的 PDF　预览程序，external　指的是外部程序
+	//"latex-workshop.view.pdf.viewer": "external",
+	//"latex-workshop.view.pdf.ref.viewer": "external",
+	//"latex-workshop.view.pdf.external.viewer.command": "D:/Users/newton/Documents/Softwares/software_windows/design/SumatraPDF-3.2-64.exe",
```

Comment:
The path here can be cleaner like `C:/path/to/my/PDFViewer.exe` - similar comments to below

Reply:
It would be useful only if you have the software Sumatra.If you don't have the software,then it is up to you to change it or remove it. Therefore I comment these ,similar comments to below.